Assets/Scripts/BeeAngryController.cs

```diff
@@ -0,0 +1,57 @@
+﻿using UnityEngine;
+using System.Collections.Generic;
+using UnityEngine.AI;
+
+public class BeeAngryController : MonoBehaviour
+{
+    public Transform player;
+    public int minDist = 2;
+
+    private NavMeshAgent agent;
+    public Animator anim;
+    private int refreshTargetTimer = 0;
+    public int refreshTargetTimerLimit = 50;
+
+    void Start()
+    {
+        anim = GetComponentInChildren<Animator>();
```

Comment:
can we call this `animator`? Just better coding practice and better for readability